app/src/components/common/BalanceRow.vue

```diff
@@ -0,0 +1,244 @@
+<template>
+  <div class="balance-row">
+    <!-- if a balance has an endTime it's for substrate undelegations -->
+    <template v-if="balance.endTime">
+      <div
+        v-if="balance.endTime"
+        :key="balance.denom + '_endtime'"
+        class="table-cell endtime"
+      >
+        <span>
+          {{ balance.endTime | fromNow }}
+        </span>
+      </div>
+    </template>
```

Comment:
This is broken:

![image](https://user-images.githubusercontent.com/40721795/91294696-38137300-e79a-11ea-9b7e-c3777d891449.png)


Please, test before pushing